pkg/apis/config/v1beta1/defaults.go

```diff
@@ -0,0 +1,234 @@
+package v1beta1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/kubeflow/katib/pkg/controller.v1beta1/consts"
+)
+
+const (
+	DefaultExperimentSuggestionName = "default"
+	DefaultMetricsAddr              = ":8080"
+	DefaultHealthzAddr              = ":18080"
+	DefaultLeaderElectionID         = "3fbc96e9.katib.kubeflow.org"
+)
+
+var (
+	DefaultInjectSecurityContext       = false
+	DefaultEnableGRPCProbeInSuggestion = true
+	DefaultWebhookPort                 = 8443
+	DefaultEnableLeaderElection        = false
+	DefaultTrialResources              = []string{"Job.v1.batch"}
+)
```

Comment:
I think we should decide where we are going to keep default values for Katib Config.
Some of them are located in this const file: https://github.com/kubeflow/katib/blob/6595c5b8e780f86c27f240d33e7917b928bb73cb/pkg/controller.v1beta1/consts/const.go#L133.
WDYT @tenzen-y @johnugeorge ?

Reply:
I'd like to locate all constants related to KatibConfig on `pkg/apis/config/v1beta1/defaults.go`.
So I'd like to move other constants such as `DefaultContainerSuggestionVolumeMountPath` to here.

Reply:
I put those constants on the `/pkg/apis/config/v1beta1/defaults.go`.
Please let me know if you have other suggestions.